README.md

```diff
@@ -23,6 +23,9 @@ Searches Elasticsearch and returns matching documents (returned document content
 
 # API
 
+**Elasticsearch:** version 5.x should be installed, not working with newer versions
```

Comment:
Remove `not working with newer versions`
"**Elasticsearch:** version 5.x should be installed" is enough